src/index.js

```diff
@@ -33,6 +33,13 @@ export default class Firedux {
     if (options.url) {
       console.warn('Firedux option "url" is deprecated, use "ref" instead.')
     }
+
+    this.v3 = options.ref !== null
+
+    if (this.v3) {
+        this.auth = firebase.auth// eslint-disable-line
```

Comment:
seems to be a version conflict 

`firebase.SDK_VERSION` = "3.2.1"
`Firebase.SDK_VERSION`= "2.4.2"